runtime/compiler/env/j9method.cpp

```diff
@@ -6902,6 +6943,36 @@ TR_ResolvedJ9Method::fieldsAreSame(I_32 cpIndex1, TR_ResolvedMethod * m2, I_32 c
    return false;
    }
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+uintptr_t
+TR_ResolvedJ9Method::memberNameAddressFromInvokeDynamicSideTable(int32_t callSiteIndex)
+   {
+   J9InvokeCacheEntry *invokeCache = (J9InvokeCacheEntry *) callSiteTableEntryAddress(callSiteIndex);
+   return (uintptr_t) invokeCache->target;
```

Comment:
This function should return address of the object reference in the table. `invokeCache->target` is the actual object reference, we should return  `&invokeCache->target`. Like `methodTypeTableEntryAddress` and `callSiteTableEntryAddress`, the return type `void *`